lib/fluent/plugin/out_forward.rb

```diff
@@ -303,21 +303,23 @@ def send_data(node, tag, chunk)
         # writeRaw(tag)
         sock.write tag.to_msgpack  # tag
 
-        # beginRaw(size)
-        sz = chunk.size
-        #if sz < 32
-        #  # FixRaw
-        #  sock.write [0xa0 | sz].pack('C')
-        #elsif sz < 65536
-        #  # raw 16
-        #  sock.write [0xda, sz].pack('Cn')
-        #else
-        # raw 32
-        sock.write [0xdb, sz].pack('CN')
-        #end
-
-        # writeRawBody(packed_es)
-        chunk.write_to(sock)
+        chunk.open(compressed: :text) do |chunk_io|
```

Comment:
This is very trivial point (because we will have another pull-request to modify here), no need to update.

This `chunk.open` SHOULD NOT have `compressed: :text` argument.
At this point of codebase, this plugin doesn't know anything about compressed chunks and buffer compression. So the plugin should behave as before by doing just `chunk.open`.
Using `chunk_io.size` instead of `chunk.bytesize` is another problem. We found that it is wrong to use `chunk.bytesize` to get the size of content in chunks, and also found that `open{|io| io.size }` is right way.